### DISCUSSION TITLE
Novice Here (again!) -  !MTUSB=1 is working to use power banks but not for extended periods as the power bank disconnects help?

### DISCUSSION THREAD

**Discussion by Tori456** [gopro/labs#746] [General]
!MTUSB=1 is working to use power banks but not for extended periods as the power bank disconnect help?

Hi! i was just wondering if there is a way to prevent the power bank from disconnecting/stopping delivering power to my go pro hero 11 black? I ideally need it to be able to be on standby for the best part of 8 hours. I have enabled the command !MTUSB=1 which has helped initially allow power banks as a power source but not the issue is just keeping it running for extended time.

Does anyone know a way to do this? or even a UK power bank suggesting of one which wont turn off after 20 mins. Thnk you in advance I really really appreciate the help I have received so far :D

**Comment by dnewman-gpsw** [comment]
Yes the problem with most power banks is they are designed charge your cell phone, and will auto shut off when the device is charged and not turn on again without a button press. This is the wrong behavior for a time lapsing or long running camera. I only use a Voltaic V50 or V75 batteries, as they are designed to be always on. You can use other batteries with a keep alive circuit like these https://www.tindie.com/products/overz/smart-power-bank-keep-alive/   https://www.tindie.com/products/plop211/power-bank-keep-alive-based-on-555-timer-kit/
